Dockerfile.android

```diff
@@ -11,9 +14,9 @@ RUN if [ -z "${mono_version}" ]; then echo -e "\n\nargument mono-version is mand
     unzip commandlinetools-linux-6609375_latest.zip && \
     rm commandlinetools-linux-6609375_latest.zip && \
     yes | tools/bin/sdkmanager --licenses && \
-    tools/bin/sdkmanager 'cmdline-tools;latest' 'build-tools;30.0.1' 'platforms;android-30' 'cmake;3.10.2.4988404'
+    tools/bin/sdkmanager --sdk_root="${ANDROID_SDK_ROOT}" 'ndk;${ANDROID_NDK_VERSION}' 'cmdline-tools;latest' 'build-tools;30.0.1' 'platforms;android-30' 'cmake;3.10.2.4988404'
 
-ENV ANDROID_SDK_ROOT=/root/sdk/
+ENV ANDROID_NDK_ROOT=${ANDROID_SDK_ROOT}/ndk/${ANDROID_NDK_VERSION}
```

Comment:
Can you move this up with the other env variables?
```
ENV ANDROID_SDK_ROOT=/root/sdk
ENV ANDROID_NDK_VERSION=21.3.6528147
ENV ANDROID_NDK_ROOT=${ANDROID_SDK_ROOT}/ndk/${ANDROID_NDK_VERSION}
```
(Also removed trailing `/` on SDK otherwise we get two `//` for NDK)

Reply:
Done!